modules/appeals_api/app/services/appeals_api/supporting_evidence/evidence_uploader.rb

```diff
@@ -1,47 +0,0 @@
-# frozen_string_literal: true
```

Comment:
I think we can keep these files around and let the `upload_complete` endpoint PR decide if it'll use them or not.